hubclient/client.go

```diff
@@ -131,23 +131,32 @@ func readBytes(readCloser io.ReadCloser, expected int64) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func validateHTTPResponse(resp *http.Response, expectedStatusCode int, debugFlags HubClientDebug) error {
+func validateHTTPResponse(resp *http.Response, debugFlags HubClientDebug, expectedStatusCodes []int) error {
```

Comment:
you can pass `expectedStatusCodes... int` instead - easier to play with if only one or 2 needs to be passed